staging/src/k8s.io/kube-gen/Godeps

```diff
@@ -239,6 +247,18 @@
 			"Rev": "f2499483f923065a842d38eb4c7f1927e6fc6e6d"
 		},
 		{
+			"ImportPath": "golang.org/x/sys/unix",
```

Comment:
Building the hyperkube binary for linux/amd64 on `master` currently comes to 216,039kB. With my changes the binary is 216,053kB. So this adds 13.4 kilobytes to the binary size. Is that acceptable?